src/sentry/auth/access.py

```diff
@@ -135,6 +140,10 @@ def project_ids_with_team_membership(self) -> FrozenSet[int]:
     def accessible_project_ids(self) -> FrozenSet[int]:
         pass
 
+    @property
+    def is_org_auth_token(self) -> bool:
```

Comment:
Is this always going to be `False`? Where/why could this be `True`?

Reply:
Actually is this supposed to be a method that returns `False` unless the subclass that inherits from this specifically changes it to return `True`?

Reply:
Yes this should always return `False`, the only one that returns `True` is the access subclass that's specific to org auth tokens